P0009/P0009.md

```diff
@@ -2305,13 +2304,15 @@ template<class StridedLayoutMapping>
 
    * [4.4]{.pnum} `StridedLayoutMapping::is_always_strided()` is `true`.
 
-[5]{.pnum} *Precondition:* 
+[5]{.pnum} *Preconditions:*
 
    * [5.1]{.pnum} `StridedLayoutMapping` meets the layout mapping requirements, 
 
-   * [5.2]{.pnum} `other.required_span_size()` is a representable value of type `size_type` ([basic.fundamental]), and
+   * [5.2]{.pnum} `other.stride(r) > 0` is `true` for all rank index `r` of `extents()`,
+
+   * [5.3]{.pnum} `other.required_span_size()` is a representable value of type `size_type` ([basic.fundamental]), and
 
-   * [5.3]{.pnum} _`OFFSET`_`(other) == 0` is `true`.
+   * [5.4]{.pnum} _`OFFSET`_`(other) == 0` is `true`.
 
 <!-- TODO static cast other.stride(d) -->
```

Comment:
Delete this comment.  The static cast thing happens in the above constructor's wording.